src/components/shop/item-catalogue.js

```diff
@@ -26,14 +26,14 @@ export const initRegistry = (key, items) => {
         return false;
     };
 
-    const itemList = {
+    const catalogueSection = {
```

Comment:
Prefer itemList for this. Does what it says on the tin and this is the end agencies will deal with.

Reply:
9b5c1cb